Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -54,20 +64,89 @@ export default function widgetBehavior(publicAPI, model) {
     if (isDragging && model.pickable) {
       return publicAPI.handleEvent(callData);
     }
+    if (onScrolling) {
+      if (model.previousPosition !== callData.position) {
```

Comment:
shouldn't you compare element per element ?

Reply:
Is it necessary ?

Reply:
well it depends what you want to compare... It is a semantical question.
```
a = {x: 1, y: 1}
b = {x: 1, y: 1}
b !== a // it is true 
c = a
c !== a // it is false
```

Reply:
Then, I do not understand how it can works now.

Reply:
it works because for each new movement, callData.position is different (pointers are different).

Reply:
Yes but that sounds weird. But, maybe in order to not being confused, I'll check the y values.